src/project-1.js

```diff
@@ -1,140 +1,115 @@
 // Do not change any of the funcxtion names
 
-const multiplyByTen = (num) => {
-  // return num after multiplying it by ten
-  // code here
-};
-
-const subtractFive = (num) => {
-  // return num after subtracting five
-  // code here
-};
-
-const areSameLength = (str1, str2) => {
-  // return true if the two strings have the same length
-  // otherwise return false
-  // code here
-};
-
-const areEqual = (x, y) => {
-  // return true if x and y are the same
-  // otherwise return false
-  // code here
-};
-
-const lessThanNinety = (num) => {
-  // return true if num is less than ninety
-  // otherwise return false
-  // code here
-};
-
-const greaterThanFifty = (num) => {
-  // return true if num is greater than fifty
-  // otherwise return false
-  // code here
-};
-
-const add = (x, y) => {
-  // add x and y together and return the value
-  // code here
-};
-
-const subtract = (x, y) => {
-  // subtract y from x and return the value
-  // code here
-};
-
-const divide = (x, y) => {
-  // divide x by y and return the value
-  // code here
-};
-
-const multiply = (x, y) => {
-  // multiply x by y and return the value
-  // code here
-};
-
-const getRemainder = (x, y) => {
-  // return the remainder from dividing x by y
-  // code here
-};
-
-const isEven = (num) => {
-  // return true if num is even
-  // otherwise return false
-  // code here
-};
-
-const isOdd = (num) => {
-  // return true if num is odd
-  // otherwise return false
-  // code here
-};
-
-const square = (num) => {
-  // square num and return the new value
-  // code here
-};
-
-const cube = (num) => {
-  // cube num and return the new value
-  // code here
-};
-
-const raiseToPower = (num, exponent) => {
-  // raise num to whatever power is passed in as exponent
-  // code here
-};
-
-const roundNumber = (num) => {
-  // round num and return it
-  // code here
-};
-
-const roundUp = (num) => {
-  // round num up and return it
-  // code here
-};
-
-const addExclamationPoint = (str) => {
-  // add an exclamation point to the end of str and return the new string
-  // 'hello world' -> 'hello world!'
-  // code here
-};
-
-const combineNames = (firstName, lastName) => {
-  // return firstName and lastName combined as one string and separated by a space.
-  // 'Lambda', 'School' -> 'Lambda School'
-  // code here
-};
-
-const getGreeting = (name) => {
-  // Take the name string and concatenate other strings onto it so it takes the following form:
-  // 'Sam' -> 'Hello Sam!'
-  // code here
-};
+const multiplyByTen = num => num * 10;
```

Comment:
Your code looks very clean.  You didn't have uneven spacing and you followed a consistent pattern and style.  This looks very professional